modules/features2d/misc/java/test/FASTFeatureDetectorTest.java

```diff
@@ -78,20 +77,24 @@ public void testDetectMatListOfKeyPointMat() {
 
     public void testEmpty() {
 //        assertFalse(detector.empty());
-        fail("Not yet implemented"); //FAST does not override empty() method
+        fail("Not yet implemented"); // FAST does not override empty() method
     }
 
     public void testRead() {
-        String filename = OpenCVTestRunner.getTempFileName("yml");
+        String filename = OpenCVTestRunner.getTempFileName("xml");
 
-        writeFile(filename, "%YAML:1.0\n---\nthreshold: 130\nnonmaxSuppression: 1\n");
+        writeFile(filename, "<?xml version=\"1.0\"?>\n<opencv_storage>\n<name>Feature2D.FastFeatureDetector</name>\n<threshold>10</threshold>\n<nonmaxSuppression>1</nonmaxSuppression>\n<type>2</type>\n</opencv_storage>\n");
```

Comment:
Why is test code changed?
Are old configurations become broken?

Reply:
The PR adds adds more parameters to saved configuration. The YAML->XML replacement looks strange, but the line changes anyway. I decided not to rework original solution.